docker/README.md

```diff
@@ -327,10 +329,40 @@ Instructions on how to build standalone docker images (for development or
 production), outside of `docker-compose`, see the
 [docker docs](../docs/DOCKER.md).
 
+### Using bitcoind version
+If you are using the bitcoind version of the compose file i.e `docker-compose-bitcoind.yml`, follow these additional instructions:
+
+#### Start Bitcoin Node with bitcoind using Docker Compose
+To launch the Bitcoin node using bitcoind in the regtest network using Docker Compose, use the following command:
+```shell
+$  NETWORK="regtest" docker-compose -f docker-compose-bitcoind.yml up
+```
+
+#### Generating RPCAUTH
+Since using `rpcuser` and `rpcpassword` is deprecated in bitcoind core we need to use `rpcauth` instead. You could generate `rpcauth` using [rpcauth.py script](https://github.com/bitcoin/bitcoin/blob/master/share/rpcauth/rpcauth.py) from bitcoin core.
+
+#### Mining in regtest using bitcoin-cli
+For mining in the regtest network, you can use `bitcoin-cli` with the following command:
+```shell
+$  bitcoin-cli -chain=regtest -rpcauth=<your_rpc_auth> generatetoaddress 10 <your_regtest_address>
```

Comment:
Since this only works within the `bitcoind` container, we should change this to:

```
$ docker exec -ti bitcoind bitcoin-cli -chain=regtest -rpcuser=devuser -rpcpassword=devpass generatetoaddress 10 2N1NQzFjCy1NnpAH3cT4h4GoByrAAkiH7zu
```

And mention that the address is just a random/dummy address and can be replaced with your own regtest address. But to get `lnd` syncing (so before you can ask it to give you a wallet address you can then send coins to), you need to mine at least 1 block. So there's a bit of a catch-22 situation. That's why it's best to just give the user a command that works for mining the first block and then later mention they can use an address from their `lnd` wallet to send themselves coins.